src/db/Client.ts

```diff
@@ -0,0 +1,21 @@
+import { ConnectionModelType } from './Models';
+
+const { Client } = require('pg');
+
+export default class PgClient {
+  static model: ConnectionModelType;
+
+  public static async newClient(): Promise<any> {
+    return new Client({
```

Comment:
We should be using a `Pool` instead of a `Client`

Reply:
This was part of my Task, so i did this in a future PR